pkgs/applications/terminal-emulators/cool-retro-term/default.nix

```
@@ -1,22 +1,21 @@
 { lib, stdenv, fetchFromGitHub, mkDerivation, qtbase, qtquick1, qmltermwidget
-, qtquickcontrols, qtgraphicaleffects, qmake, nixosTests }:
+, qtquickcontrols, qtquickcontrols2, qtgraphicaleffects, qmake, nixosTests }:
 
 mkDerivation rec {
-  version = "1.1.1";
   pname = "cool-retro-term";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "Swordfish90";
     repo = "cool-retro-term";
     rev = version;
-    sha256 = "0mird4k88ml6y61hky2jynrjmnxl849fvhsr5jfdlnv0i7r5vwi5";
+    sha256 = "PewHLVmo+RTBHIQ/y2FBkgXsIvujYd7u56JdFC10B4c=";
   };
 
-  patchPhase = ''
-    sed -i -e '/qmltermwidget/d' cool-retro-term.pro
-  '';
+  patches = [ ./use-system-qmltermwidget.patch ];
 
-  buildInputs = [ qtbase qtquick1 qmltermwidget qtquickcontrols qtgraphicaleffects ];
+  buildInputs = [ qtbase qtquick1 qmltermwidget qtquickcontrols2
+    qtgraphicaleffects ] ++ lib.optional stdenv.isDarwin qtquickcontrols;
```

```suggestion
  buildInputs = [
    qtbase
    qtquick1
    qmltermwidget
    qtquickcontrols2
    qtgraphicaleffects
  ] ++ lib.optional stdenv.isDarwin qtquickcontrols;
```
this was a bit weird, lets do it properly.

---

I'll make this change -- is there a style guide or formatter for nix lang, by the way?

---

https://github.com/nix-community/nixpkgs-fmt